src/WarcraftLegacies.Source/Quests/Scourge/QuestLichKingArthas.cs

```diff
@@ -14,13 +14,15 @@ public sealed class QuestLichKingArthas : QuestData
   {
     private readonly unit _utgardeKeep;
     private readonly Artifact _helmOfDomination;
+    private readonly QuestData _questToCompleteUponCompletion;
 
-    public QuestLichKingArthas(unit utgardeKeep, Artifact helmOfDomination) : base("The Ascension",
+    public QuestLichKingArthas(unit utgardeKeep, Artifact helmOfDomination, QuestData questToCompleteUponCompletion) : base("The Ascension",
```

Comment:
Let's just remove the entire `QuestCivilWar` quest entirely, and make it so that `QuestAscension` does not change your team.